content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+
+2. **Service**: It uses NodePort to expose the `hello-universe` application on each Node at a random port between 30000-32767. 
+<br />
+
+
+If there are no errors or compatibility issues, Palette displays the newly created full cluster profile. Verify the layers you added, and click on the **Next** button. 
+<br />
+
+### Review
+Review all layers and click **Finish Configuration** to create the cluster profile. 
+<br />
+
+## Create a Cluster
+Click on the newly created cluster profile to view its details page. Palette displays all the layers again and allows you to edit any of them. Click on the **Deploy** button and refer to the [Create an Edge Native Host Cluster](/clusters/edge/site-deployment/site-installation/cluster-deployment) guide to deploy a new Edge cluster. 
+<br />
+
+![Screenshot of the Profile Layers success.](/tutorials/edge-native/clusters_edge_deploy-cluster_profile-success.png)
+
+The cluster deployment wizard will display the following sections. 
+<br />
+
+### Basic Information
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Cluster name| docs-tutorial-cluster |
+|Description| Cluster as part of the Edge tutorial.|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:false`|
+
+Click **Next** to continue.
+<br /> 
+
+### Parameters
+The **Parameters** section offers you another opportunity to change the profile configurations. For example, clicking on the **BYOS Edge OS 1.0.0** layer allows you to configure the `system.registry`, `system.repo`, and other available attributes. 
+
+Use the default values for all attributes across all layers and click **Next**.
+<br />
+
+### Cluster config
+Provide the Virtual IP (VIP) address for the host cluster to use during the cluster configuration process. You can optionally select an SSH key to remote into the host cluster and provide a Network Time Protocol (NTP) server list, if available, with you.  
+
+Click **Next** to continue.    
+<br />
+
+### Nodes config
+In this section, provide details for the master pool. 
+
+|**Field** | **Value for the master-pool**| 
+|---| --- | 
+|Node pool name| master-pool | 
+|Allow worker capability| Checked | 
+|Additional Labels (Optional) | None |
+|[Taints](/clusters/cluster-management/taints/)|Off|
+|Pool Configuration > Edge Hosts | Choose one of the registered Edge hosts.<br />Palette will automatically display the Nic Name for the selected host. |
+
+The screenshot below shows adding an Edge host to the master pool.
+
+![Screenshot of adding an Edge host to the master pool.](/tutorials/edge-native/clusters_edge_deploy-cluster_add-master-node.png)
+
+
+Similarly, provide details for the worker pool, and add the remaining two Edge hosts to the worker pool. If you do not have Edge hosts remaining, remove the worker pool. Your master pool has the worker capability already. 
```

Comment:
Let's add a screenshot here too. This is perhaps one of the most important part of the flow so it warrants an image.